client/src/cbltest/api/testserver.py

```diff
@@ -117,3 +144,26 @@ async def log(self, msg: str) -> None:
             TestServerRequestType.LOG, payload
         )
         await self.__request_factory.send_request(self.__index, request)
+
+    def __extract_variant(self, info: GetRootResponse) -> ServerVariant:
```

Comment:
Is it not possible to make this a method on GetRootResponse?

Reply:
Yes, I refactored based on this.